Cargo.toml

```diff
@@ -41,5 +41,6 @@ default = ["proxy", "use-rustls"]
 minimal = []
 debug-calls = []
 proxy = ["byteorder", "winapi", "libc"]
-use-rustls = ["webpki-roots", "rustls"]
+use-rustls = ["webpki-roots", "rustls/default"]
+use-rustls-ring = ["proxy", "webpki-roots", "rustls/ring", "rustls/logging", "rustls/std", "rustls/tls12"]
```

Comment:
@thunderbiscuit Do we need the proxy feature to be set when using `use-rustls-ring` ?
Is this just to keep the standard like the `default` ones ?

Reply:
Yep you're right. Fixed in my latest push.